### DISCUSSION TITLE
请问模型可以支持Function Calling吗？

### DISCUSSION THREAD

**Discussion by ctruk** [zai-org/CogVLM2#55] [General]
Function Calling功能很实用，可以大大扩展模型能力。加上这个模型本身的多模态能力，能碰撞出更多可能性。

**Comment by zRzRzRzRzRzRzR** [comment]
这个模型的基座是llama3-8b-instruct，我们训练数据没有fc，但是你可以尝试直接调用这个模型基座能力的fc

**Reply by ctruk** [reply]
我查了下，llama3应该支持FC的，有人实践过：https://www.zhihu.com/question/653373790/answer/3482143180
工程是这个：https://github.com/smalltong02/keras-llm-robot/tree/main
所以我想llama3基座支持的话，CogVLM2是不是也有机会改进支持。

**Reply by zRzRzRzRzRzRzR** [reply]
但是我们训练中没有加入这样的数据，而是主要以图片理解为主，不是一个融合的模型
